contributing/metadata.md

```diff
@@ -0,0 +1,166 @@
+# Contributing Metadata Types to the Registry
+
+> NOTE: If you have a “Settings” metadata type, you do not need to follow these instructions. Support for your type is already built into SDR.
+
+## Adding new types to the registry via a script
+
+To simplify modifying the registry, there's a script.
+
+> Note: The script depends on being able to create an org that will have your type. If you have a working org but your type isn't compatible with scratch org, you can [use an existing org](#use-an-existing-org)
+
+1. looks for missing types (similar to the completeness test)
+2. For missing types, generate a project and scratch org that includes the Features/Settings
+3. Running force:mdapi:describemetadata to get the describe
+4. Modifying the registry to include the newly found types
+
+```shell
+yarn update-registry
+```
+
+### What the script can't do
+
+inFolderTypes and types with childXml in their describe are not supported. You **want** to explore the various strategies for those (see the SDR Handbook in this repo) and create NUTs to validate that your types are being handled correctly.
```

Comment:
```suggestion
`inFolderTypes` and types with `childXml` in their describe are not supported. You **want** to explore the various strategies for those (see the [SDR Handbook](../HANDBOOK.md) in this repo) and create NUTs to validate that your types are being handled correctly.
```

:question: Based on the wording, it seems like "describe" is an element. If so, that should probably be back-ticked as well.
:question: It might help for "create NUTs" to link to the *Integration Testing* header?

:warning: Confirm the relative URL for the Handbook.